homeassistant/components/livisi/coordinator.py

```diff
@@ -84,7 +84,7 @@ async def async_get_devices(self) -> list[dict[str, Any]]:
 
     async def async_get_pss_state(self, capability: str) -> bool | None:
         """Set the PSS state."""
-        response: dict[str, Any] = await self.aiolivisi.async_get_pss_state(
+        response: dict[str, Any] = await self.aiolivisi.async_get_device_state(
```

Comment:
How is this linked to reauth?

Reply:
Here is only a rename for this method and I added it in the 0.0.15 version of aiolivisi.

Reply:
I will create a new pull request for this, thank you!

Reply:
The new pull request with the dependency bump is here: https://github.com/home-assistant/core/pull/86721